packages/protocol/deployments/mainnet-deploy-logs.md

```diff
@@ -0,0 +1,238 @@
+# Taiko Mainnet Contracts
+
+## L1 Contracts
+
+### shared_address_manager
+
+- proxy: `0xEf9EaA1dd30a9AA1df01c36411b5F082aA65fBaa`
+- impl: `0x9cA1Ab10c9fAc5153F8b78E67f03aAa69C9c6A15`
+- owner: `0x56706F118e42AE069F20c5636141B844D1324AE1`
+- pendingOwner: `admin.taiko.eth`
+- todo:
+  - admin.taiko.eth to accept the owner
+- logs:
+  - deployed on May 1, 2024 at commit `56dddf2b6`
+
+### taiko_token
+
+- proxy: `0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800`
+- impl: `0x9ae1a067f9655dd0511390e3d70bb25933ae61eb`
+- owner: `labs.taiko.eth`
+- logs:
+  - deployed on April, 2024 at commit `?????`
```

Comment:
@davidtaikocha do you still remember the commit for Taiko Token contract deployed?

Reply:
Ideally we also create and maintain such a deploy/upgrade log for hekla.

Reply:
After comparing the verified source code, its on: `2f6d3c62e`